source/User_Guide/email_activity_beta.md

```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
+* Troubleshoot email delivery issues faster with comprehensive and sequential event data by email.
+* Pinpoint specific emails easily with advanced search and filter options --including subject line and email metadata.
 * Customize your data independently with CSV download/export option.
 * Increase storage available for up to 60 days for historical visibility within the UI by purchasing an upgrade.
-* Complete an API call to gather specific email events when additional storage is purchased.
+* Call all events by an email from the API with additional storage purchase.
 
 {% info %}
-SendGrid stores two days of Email Activity data for free. To increase your storage capability, see LINK
+SendGrid stores two days of Email Activity data for free. To increase your storage capability, you may purchase more storage [here](https://app.sendgrid.com/settings/billing).
 {% endinfo %}
 
-{% anchor h2 %}
+{% anchor H2 %}
+Types of email activity data
+{% endanchor %}
+
+The Email Activity feed allows you to view specific information about messages sent from your account. 
+
+
+<table class="table" style="table-layout:fixed">
+  <tr>
+  <th class="tg-031e">Event</th>
+    <th class="tg-yw4l">Description</th>
+    </tr>
+    <tr>
+    <td class="tg-031e">Processed</td>
+    <td class="tg-yw4l">Requests from your website, application, or mail client via SMTP Relay or the API that SendGrid processed.</td>
+  </tr>
+  <tr>
+    <td class="tg-031e">Clicks</td>
+    <td class="tg-yw4l">Whenever a recipient clicks one of the Click Tracked links in your email.</td>
+  </tr>
+  <tr>
+    <td class="tg-031e">Delivered</td>
+    <td class="tg-yw4l">An email that was delivered to a recipient.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Opens</td>
+    <td class="tg-yw4l">Whenever an email is opened by a recipient.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Unsubscribes</td>
+    <td class="tg-yw4l">Whenever a recipient unsubscribes from your emails. </td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Group Unsubscribes</td>
+    <td class="tg-yw4l">Whenever a recipient unsubscribes from a suppression group.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Group Resubscribes</td>
+    <td class="tg-yw4l">When a recipient resubscribes themselves to a suppression group.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Deferred</td>
+    <td class="tg-yw4l">The recipient mail server asked SendGrid to stop sending emails so fast.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Drops</td>
+    <td class="tg-yw4l">SendGrid will drop an email when the contact on that email is in one of your suppression groups, the recipient email previously bounced, or that recipient has marked your email as spam.</td>
+  </tr>
+  <tr>
+    <td class="tg-yw4l">Bounces </td>
+    <td class="tg-yw4l">When an email is attempted to be delivered, but the recipient mail server rejects it.</td>
```

Comment:
The receiving server could not or would not accept the message. If a recipient has previously unsubscribed from your emails, your attempt to send to them is bounced.